deepspeed/launcher/constants.py

```diff
@@ -8,6 +8,7 @@
 
 OPENMPI_LAUNCHER = 'openmpi'
```

Comment:
On CPU, we support MPICH and Intel MPI (IMPI) launcher.

Reply:
Is there any issue to resolve here? Or was this just for documentation?

Reply:
just documentation, will mark as resolved.